lib/handler/chunked.c

```diff
@@ -74,6 +74,9 @@ static void on_setup_ostream(h2o_filter_t *self, h2o_req_t *req, h2o_ostream_t *
 {
     chunked_encoder_t *encoder;
 
+    if (h2o_is_subrequest(req))
```

Comment:
I would appreciate it if you could leave a TODO comment here, stating that http1.c should decide whether chunked encoding should be applied rather than having a separate module to determine if the underlying transport is in fact HTTP/1.1.